sponsor.html

```diff
@@ -0,0 +1,216 @@
+              <li>Short (~5 min.) pre-recorded presentation at the start of the competition, to talk about your company and set your challenge, and a short in-person presentation on the final weekend to announce the winner</li>
+            </ul>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column d-12-12 sponsor-level">
+          <p style="margin-top: 1rem;">
+            If there is anything not mentioned we can offer you or you would be interested in?? Get in touch!
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column d-12-12 sponsor-level">
+          <p style="margin-top: 1rem;">
+	    Please register to become a sponsor, and to help shape the future of these young engineers.
+            <img src="{{ '/images/advertising/sad-robot.jpg' | prepend: site.baseurl }}" />
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width text-center">
+      <a name="signup"></a><h2>Want to sponsor us?</h2>
+      <div class="column m-8-12 m-offset-2-12 text-left">
+        <p>
+          Please fill in the form below to and we will get in touch with you soon!
+        </p>
+        <!--TODO: Create sponsor sign up form-->
+        {% include sponsor-signup.html %}
+      </div>
+    </div>
+  </div>
+</div>
+<!--TODO: Update the FAQ to be about sponsorhip-->
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <a name="faq"></a><h2 class="text-center">Frequently Asked Questions</h2>
+      <div class="row">
+        <div class="column m-6-12">
+          <p class="question">Who can sponsor?</p>
+          <p>
+	    Any individual or organisation is able to sponsor Student Robotics.
+          </p>
+          <p class="question">Why do we need sponsors</p>
+          <p>
+            We do not charge the teams in order to enter the competition.
+	    However there are still costs associated in running the competition
+	    , and as such we depend upon commercial sponsors in order to keep
+	    running. Additionally, the robotics kits that are loaned to teams
+	    require ongoing maintenance and repairs. Thanks to our sponsors we 
+	    are able to continue to offer this at no cost to the competitors.
+	  </p>
+          <p class="question">Can I add Gift Aid to my donation?</p>
+          <p>
+            If you are a UK tax payer and eligible to give Gift Aid then you
+	    can add Gift Aid to your donation. Please donate through the
+	    JustGiving link above and tick the Gift Aid box.
+          </p>
+          <p class="question">How does sponsoring an award work?</p>
+          <p>
+	    Each year we present 5 awards to teams based on their involvement
+	    throughout the year, and not based on the competition winners.
+	    Details of these awards can be found in the current rulebook.
```

Comment:
Realisitically we are unlikely to be attracting sponsors for the current year, but instead the next one. As such the current rulebook will not be massively relevant, however these awards do need to be referenced in some way to avoid annoying interested sponsors.

My feeling is that putting details of the individual awards on this page will bloat it and detract from the point of "Please give us money", as well as giving us an extra thing to update each year, so linking to the rulebook is probably better.

Reply:
Link added in below commit by @antoinepetty